styles/variables-gallium.less

```diff
@@ -90,6 +90,41 @@
 @agate-header-subtitle-font-size: 27px;
 @agate-header-subtitle-font-weight: normal;
 
+// Heading
+// ---------------------------------------
+@agate-heading-border-width: 3px;
+@agate-heading-padding: 24px;
+@agate-heading-showline-padding: 12px;
+@agate-heading-letter-spacing: normal;
+@agate-heading-line-height: 1.6em;
+@agate-heading-title-font-family: @agate-base-font-family;
+@agate-heading-title-font-size: 70px;
```

Comment:
Should this font size be larger than `@agate-heading-large-font-size` (48px)?

Reply:
Indeed it should! Title -> SubTitle -> Large -> Medium -> Small -> Tiny, which relate to `h1` -> `h6`. Clever eh? :D